andela/ah-the-jedi-backend
# #165305251 Upgrade Django app from Django 1.10 to Django 2.1

### What does this PR do?
Upgrade Django from version 1.10 to 2.1

### Description of Task(s) to be completed?

Upgrade Django from v1.10 to v2.1
Upgrade all other django extensions and django rest to latest versions
Update url patterns to include features included in Django 2.1 that were not in Django 1.0

### How should this be manually tested?

Clone this repository on local machine and CD into project folder

`$ git clone git@github.com:andela/ah-the-jedi-backend.git`
`$ cd ah-the-jedi-backend`

Check out to the PR's branch

`$ git checkout ch-upgrade-django-165305251`

Create and activate virtual environment

`$ python3 -m venv venv`
`$ source .venv/bin/activate`

Install app dependencies from requirements.txt

`$ pip install -r api/requirements.txt`

Check the versions of the installed packages

`$ pip freeze`

### What are the relevant Pivotal Tracker stories?
[165305251](https://www.pivotaltracker.com/story/show/165305251)

### Screenshots

![image](https://user-images.githubusercontent.com/11965186/56574772-bff8d400-65cc-11e9-913e-57aca0408af0.png)
